src/main/scala/com/databricks/spark/xml/util/TypeCast.scala

```diff
@@ -268,4 +277,17 @@ private[xml] object TypeCast {
       TypeCast.castTo(data, FloatType, options).asInstanceOf[Float]
     }
   }
+
+  private[xml] def isParseableAsZonedDateTime(value: String,
```

Comment:
For later, we might have to figure out a way to cache the parsing of a custom format into a pattern, and maybe this check, because it'll happen for every single row

Reply:
I should remove the parsing error check, as it should be caught in `parseXmlTimestamp()` method